src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -135,10 +137,15 @@ protected override Task UpdateDisplayAsync(in RenderBatch batch)
 
                 var renderId = Interlocked.Increment(ref _nextRenderId);
 
+                var valueStopWatch = _logger.IsEnabled(LogLevel.Debug) ?
+                    ValueStopwatch.StartNew() :
+                    (ValueStopwatch?)null;
```

Comment:
Do we do this other places we use `ValueStopwatch`? I thought they were really cheap to create.

Reply:
I guess you're right. I'll change this.